devlab/jenkins/run_functional_tests.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ -z "$WORKSPACE" ]; then
+    echo "'WORKSPACE' is not defined"
+    exit 1
```

Comment:
Once again ... in this case we can't run this script from devlab manually. It's simple to define it if not defined.